app/controllers/api/api.js

```diff
@@ -18,4 +22,60 @@ function authenticatedUser(req, res, next) {
 
 router.get('/secret', authenticatedUser, function (req, res, next) {
   res.json({message: "secret"});
-});
\ No newline at end of file
+});
+
+//INDEX
+router.get('/api/knits', function (req, res){
+  Knit.find({}, function (err, knits){
+    if (err) {
+      res.send("There was an error with your GET request " + err)
+    } else {
+      res.send(knits);
+    }
+  });
+})
+
+//CREATE
+router.post('/api/knits', authenticatedUser, function (req, res){
+  Knit.create(req.body.knit, function (err, knit){
+    console.log(req.body.knit);
+    if (err) {
+      res.send("There was an error with your POST request " + err)
+    } else {
+      res.redirect('/api/knits');
```

Comment:
since you are going to POST from the client side, you shouldn't redirect, but send a json to tell the client that everything went fine:
```js
res..json({success: true})
```